tools/generators/pbxtargetdependencies/src/Generator/ConsolidationMapsArguments.swift

```diff
@@ -379,7 +379,7 @@ extension ConsolidationMapsArguments {
 
 private extension Array {
     func slicedBy<CountsCollection>(
-        targetIndex: Array<TargetID>.Index,
+        targetIndex: Int,
```

Comment:
Does this work without having to update the call sites?

Reply:
Yeah. All `Array` indexes are `Int`.